scripts/update-integrations.mjs

```diff
@@ -181,4 +187,40 @@ Updated: ${existingIntegrations.size - deprecatedIntegrations.length} integratio
 	console.info(stats)
 }
 
-main()
+async function safeUpdateExistingIntegrations() {
+	const pathname = path.resolve(
+		path.dirname(fileURLToPath(import.meta.url)),
+		"../src/content/integrations/*.md",
+	)
+	const entries = await glob(pathname)
+
+	for (const entry of entries) {
+		const { data } = matter.read(entry)
+
+		// only override NPM download stats for safe updates
+		const downloads = await fetchDownloadsForPackage(data.name)
+
+		const frontmatter = yaml.stringify({
+			...data,
+			downloads,
+		})
+
+		delete frontmatter.badges
+
+		fs.writeFileSync(
+			entry,
+			`---
+${frontmatter}---\n`,
+		)
+	}
+}
+
+const args = process.argv.slice(2)
+
+// only fetch unsafe changes like new and deprecated integrations
+// if the --unsafe CLI flag was provided
+if (args.includes("--unsafe")) {
```

Comment:
This flag is included in the weekly GitHub Action only, nightly will run safe migrations only